src/main/java/randoop/main/Minimize.java

```diff
@@ -1190,10 +1188,10 @@ private static void addImport(CompilationUnit compilationUnit, String importName
   /**
    * Sort a compilation unit's imports by name.
    *
-   * @param compilationUnit the compilation unit whose imports will be sorted by name
+   * @param compUnit the compilation unit whose imports will be sorted by name
```

Comment:
Is this change necessary?  I found the previous name clearer and not excessively long.

Reply:
I was trying to make the variable names more consistent. I will change them all to be compUnit.